src/OrchardCore/OrchardCore.Abstractions/IClientIPAddressAccessor.cs

```diff
@@ -1,8 +1,6 @@
-using System.Net;
-
 namespace OrchardCore;
 
 public interface IClientIPAddressAccessor
 {
-    Task<IPAddress> GetIPAddressAsync();
+    Task<string> GetIPAddressAsync();
```

Comment:
Breaking-changes

Reply:
It's also unnecessary. `IPAddress` is the type to store IP addresses. It can very well be instantiated from the header value too.

Reply:
there is another situation: the client user can pass through his  proxy server  in his area, then visit our OC  server .

Reply:
Why would that be any different, though? In the end, we always just need a single IP address.